src/olympia/users/views.py

```diff
@@ -336,7 +336,7 @@ def _clean_next_url(request):
     gets = request.GET.copy()
     url = gets.get('to', settings.LOGIN_REDIRECT_URL)
 
-    if not is_safe_url(url, host=request.get_host()):
```

Comment:
for future reference, I don't think get_host() is safe because it [reads request headers](https://docs.djangoproject.com/en/1.9/ref/request-response/#django.http.HttpRequest.get_host). We have [settings.DOMAIN](https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L183) for this purpose.